.github/workflows/inspektor-gadget.yml

```diff
@@ -1473,6 +1473,84 @@ jobs:
         gadget_tag: ${{ steps.set-repo-determine-image-tag.outputs.gadget-tag }}
         test_summary_suffix: "aro"
 
+  test-integration-eks:
+    name: Integration tests on EKS
+    # level: 4
+    needs:
+      - test-unit
+      - build-clients
+      - build-ig
+      - build-gadget-container-images
+      - publish-gadget-images-manifest
```

Comment:
(Just for my understanding) We don't need any conditional checks for this job because it will skipped for PRs because of `publish-gadget-images-manifest`/`build-gadget-container-images` ? Also Is there a reason to include `build-ig` here?

Reply:
> We don't need any conditional checks for this job because it will skipped for PRs because of publish-gadget-images-manifest/build-gadget-container-images ?

Right, it's skipped because dependencies are skipped for PRs as well. Regarding `build-ig`, 

> Also Is there a reason to include build-ig here?

I think you have a point, ig is not needed for the k8s tests, however it's implicitly needed because `build-and-push-gadgets` needs it. I'll remove it from different jobs around that I copy and pasted from.